### PR TITLE
[Chore] Fix installation instructions and docker compose

### PR DESCRIPTION
**Pull request by rameziophobia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ docker compose -f docker-compose-dev.yml build
 docker compose -f docker-compose-dev.yml up -d
 docker compose -f docker-compose-dev.yml run web bin/rails db:create # ignore any errors here
 docker compose -f docker-compose-dev.yml run web bin/rails db:migrate db:seed
+docker compose -f docker-compose-dev.yml run web bin/rails assets:precompile
```
Comment (rameziophobia) [inline README.md:25]: without precompiling the assets, tailwind will not work so it's essential to have this step.

File: docker-compose-dev.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ services:
     command: bin/rails server -b 0.0.0.0
     environment:
       DB_USERNAME: postgres
-      DB_PASSWORD: postgres
+      POSTGRES_PASSWORD: postgres
```
Comment (rameziophobia) [inline docker-compose-dev.yml:25]: The ENV variables to read db credentials changed to POSTGRES_PASSWORD but this was not updated

File: README.md (hunk 1 above)
Comment (fermion) [reply README.md:25]: Were you not seeing assets when running this locally? This should already be being executed during the build [from the Dockerfile.dev](https://github.com/goinvo/staffplan_redux/blob/4648d622a7dd29629269fd2fc4d91dfaaaa93951/Dockerfile.dev#L36).

Comment (fermion) [reply README.md:25]: Scratch what I said, this is 💯 needed. I must have had a previous build on my filesystem while I put this together 🙇🏻

**Comment by fermion** [comment]
@rameziophobia I'm going to take this PR and run with it, adding a few more things. Specifically I neglected to provide any instructions for bootstrapping for a Stripe account which is needed in order for the everything to function properly. Thank you for the contribution thus far!

**Comment by rameziophobia** [comment]
Thanks!

**Comment by fermion** [comment]
Closing in lieu of https://github.com/goinvo/staffplan_redux/pull/264 🙇🏻 for your contribution @rameziophobia!

**Pull request closed without merging**